src/NzbDrone.Core/Notifications/NotificationService.cs

```diff
@@ -189,6 +191,50 @@ public void Handle(SeriesRenamedEvent message)
             }
         }
 
+        public void Handle(EpisodeFileDeletedEvent message)
+        {
+            var deleteMessage = new EpisodeDeleteMessage();
+            deleteMessage.Message = GetMessage(message.EpisodeFile.Series, message.EpisodeFile.Episodes, message.EpisodeFile.Quality);
+            deleteMessage.Series = message.EpisodeFile.Series;
+            deleteMessage.EpisodeFile = message.EpisodeFile;
+            deleteMessage.Reason = message.Reason;
+
+            foreach (var notification in _notificationFactory.OnDeleteEnabled())
```

Comment:
This just occurred to me, should we separate series and episode file deletion events? That way you can get notified about one or the other only?

Reply:
My thoughts were to notify for both as my thoughts are if you want to know about one deletion you likely want to know about the other as well (but that was just my personal preference to be honest) - Happy to separate out if that is what is wanted though